src/qt/region.cpp

```diff
@@ -93,9 +94,36 @@ wxRegion::wxRegion(const wxBitmap& bmp)
         m_refData = new wxRegionRefData( QRect( 0, 0, bmp.GetWidth(), bmp.GetHeight() ) );
 }
 
-wxRegion::wxRegion(const wxBitmap& WXUNUSED(bmp), const wxColour& WXUNUSED(transp), int WXUNUSED(tolerance))
+wxRegion::wxRegion(const wxBitmap& bmp, const wxColour& transp, int tolerance)
 {
-    wxMISSING_IMPLEMENTATION( __FUNCTION__ );
+    if(!bmp.GetHandle()) {
```

Comment:
Formatting should be `if ( !bmp.GetHandle() )`. The same goes for all following occurrences, and `for`.
`{` should be on its own line. The same goes for all following occurrences.